test/StakeManagement.test.ts

```diff
@@ -226,9 +228,9 @@ contract('StakeManagement', function ([_, relayManager, anyRelayHub, owner, nonO
       })
 
       it('should not allow to penalize hub', async function () {
-        await expectRevert.unspecified(
-          relayHub.penalize(relayManager, nonOwner, { from: anyRelayHub }),
-          'hub authorization expired'
+        await expectRevert(
```

Comment:
This test will change in a subsequent PR. Because the RelayHub code has a bug in penalize()